apps/molgenis-components/src/client/client.js

```diff
@@ -95,9 +95,9 @@ const fetchTableData = async (
 ) => {
   const url = graphqlURL ? graphqlURL : "graphql";
   const limit =
-    properties && properties.hasOwnProperty('limit') ? properties.limit : 20;
+    properties && Object.prototype.hasOwnProperty.call(properties, 'limit') ? properties.limit : 20;
```

Comment:
why this change?

Reply:
https://learn.snyk.io/lessons/prototype-pollution/javascript/